modules/xmpp/TraceablePeerConnection.js

```diff
@@ -392,6 +392,13 @@ TraceablePeerConnection.prototype.setRemoteDescription
     description.sdp = this.rtxModifier.implodeRemoteRtxSsrcs(description.sdp);
     this.trace('setRemoteDescription::postTransform (implodeRemoteRtxSsrcs)', dumpSDP(description));
 
+    if (this.session.room.options.preferH264) {
```

Comment:
AFAIS options are kinda global for the library, so you can add it to the init section here: https://github.com/jitsi/lib-jitsi-meet/blob/master/doc/API.md#jitsimeetjs

I noticed `disableRtx` is missing there. Can you add it too?